src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -2,7 +2,7 @@ import { FC, useState, useEffect } from 'react';
 import classNames from '@/components/availability-panel/drag-drop-context/styles.module.scss';
 import { DragDropContext, DropResult } from 'react-beautiful-dnd';
 import { dragDropProps } from '@/interfaces/availabilityPanel.type';
-import { toast } from 'react-toastify';
+import toast, { ToastTypes } from '@/helperFunctions/toast';
```

Comment:
Probably good to avoid default export here and export it as const isn't it @sumitd94 @whyDontI ?